client/app/styles/_variables.scss

```diff
@@ -32,8 +32,8 @@ $terminal-header-height: 44px;
 
 $node-highlight-fill-opacity: 0.1;
 $node-highlight-stroke-opacity: 0.4;
-$node-highlight-stroke-width: 2;
-$node-border-stroke-width: 6;
+$node-highlight-stroke-width: 0.04;
+$node-border-stroke-width: 0.12;
```

Comment:
Yup, tried it and it works fine - I think the issue with small units only concerned the fonts i.e. text, no?